art/attacks/evasion/auto_projected_gradient_descent.py

```diff
@@ -275,7 +215,24 @@ def __call__(self, y_true, y_pred):
                             "the estimator has to to predict logits."
```

Comment:
Please change line 198 from `loss_object=self._loss_object,` to `loss_object=_loss_object_tf,`